actions/FirefallClient.js

```diff
@@ -87,7 +99,7 @@ class FirefallClient {
       return response;
     } catch (error) {
       logger.error('Failed feedback request:', error);
```

Comment:
Let's modify the logger statement to `'Failed sending feedback:'` to sound more natural.